ballsdex/core/bot.py

```diff
@@ -417,6 +417,10 @@ async def send(content: str):
                 )
                 # still including traceback because it may be a programming error
 
+            if isinstance(error, app_commands.TransformerError):
+                await send("One of the arguments provided cannot be parsed.")

```

Comment:
I think it's better to log these errors, it's good to acknowledge but errors should still not happen and be handled before.
```suggestion
                await send("One of the arguments provided cannot be parsed.")
                log.error(f"Failed running converter", exc_info=error.original)
```

Reply:
I'm not sure if we want to log these as it's just stuff failing the default model converters. I believe our custom transformers handle their own errors